src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -257,12 +265,23 @@ class TableView extends React.Component<TableViewProps> {
       return dataRow[column.key];
     }
 
-    const hasLinkField = eventView.hasAutolinkField();
-    const forceLink =
-      !hasLinkField && eventView.getFields().indexOf(String(column.field)) === 0;
+    // TODO: remove this
+    // const hasLinkField = eventView.hasAutolinkField();
+    // const forceLink =
+    //   !hasLinkField && eventView.getFields().indexOf(String(column.field)) === 0;
+    const forceLink = false;
```

Comment:
If we don't force a link how do users get to the details view?

Reply:
@markstory I think we'll need to change this. I'll bring this up with Dora and Bowen.

/cc @doralchan @bowencai8

Reply:
I've addressed this in https://github.com/getsentry/sentry/pull/15793/commits/66851e26574100845f3a3f6bc6b165bf8cb8846b